aviary/docs/examples/reserve_missions.ipynb

```diff
@@ -98,7 +97,6 @@
     "\n",
     "from aviary.examples.reserve_missions.run_reserve_mission_fixedrange import phase_info\n",
     "\n",
-    "check_phase_info(phase_info, HEIGHT_ENERGY)\n",
```

Comment:
I guess we don't need `check_phase_info` at all anymore because if it doesn't match with the library definition it's rejected?

Reply:
We don't need it to run in `check_and_preprocess_inputs` anymore, because the args are checked against the phase builders options dictionary. There may be a secondary use for it, just for making sure the doc phase_infos are correct though.